aws_tools/s3_handler.py

```diff
@@ -41,7 +41,7 @@ def download_file(self, key, local_file):
         :param string key: object to download
         :param string local_file: file to download to
         """
-        body = self.resource.Object(self.bucket_name, key).get()['Body']
+        body = self.resource.Object(bucket_name=self.bucket_name, key=key).get()['Body']
```

Comment:
Sorry, been too long in Java land.  Of course Python and Ruby support named parameters.